docs/docs-source/docs/modules/get-started/pages/install-cloudflow.adoc

```diff
@@ -0,0 +1,322 @@
+
+broker-1-address:broker-1-port, broker-2-address;broker-2-port
+
+If you want to test Cloudflow and need a Kafka cluster, we recommend using Strimzi, a third-party Kafka operator that can create and manage Kafka clusters. 
+
+The following chapter is a guide on how to configure and install a Kafka cluster using Strimzi. 
+
+. xref:install-and-use-strimzi.adoc[How to install Strimzi and create a Kafka cluster]
+
+=== Storage requirements
+
+If you plan to write applications using Spark or Flink, the Kubernetes cluster will need to have a storage class of the `ReadWriteMany` type installed. The name of the `ReadWriteMany` storage class name is needed when installing Cloudflow.
+
+== Installing Cloudflow
+
+In this guide, we will use Helm to install Cloudflow.
+
+The first step is to create the namespace we want to install Cloudflow into:
+
+  kubectl create ns cloudflow
+
+IMPORTANT: Many subsequent commands will assume that the namespace is `cloudflow`.
+
+First, we add the Cloudflow Helm repository:
+
+  helm repo add cloudflow-helm-charts https://lightbend.github.io/cloudflow-helm-charts/
+
+The `repo update` command is to make sure Helm has an up-to-date index of all Helm charts.
+
+  helm repo update
+
+Now we install Cloudflow, but before you press enter, let's go over the custom parameter we are supplying with the command:
+
+  cloudflow_operator.persistentStorageClass="nfs"
+
+The `persistentStorageClass` parameter sets the storage class to use with Spark and Flink applications. This storage class, as previously mentioned, has to be of the type `ReadWriteMany`. In our example, we are using the `nfs` storage class, which we have installed in our cluster using the https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner[NFS Server Provisioner Helm chart].
+
+  cloudflow_operator.kafkaBootstrapservers=cloudflow-strimzi-kafka-bootstrap.cloudflow:9092
+
+The `kafkaBootstrapservers` parameter sets the address and port of the Kafka cluster that Cloudflow will use. In this example, we have used the address of a Strimzi created Kafka cluster located in the `cloudflow` Kubernetes namespace.
+
+Installing Cloudflow using the Helm chart.
+
+  helm install cloudflow cloudflow-helm-charts/cloudflow --namespace cloudflow --set
+  strimzi.enabled=true --set cloudflow_operator.persistentStorageClass="nfs" --set cloudflow_operator.kafkaBootstrapservers=cloudflow-strimzi-kafka-bootstrap.cloudflow:9092
+
+Check the status of the install process using `kubectl`. When the Cloudflow operator pod is in `Running` status, the installation is complete.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS    RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running   0          10s
+----
+
+You can now deploy an Akka based Cloudflow application into the cluster. More on this in the development section of the documentation.
+
+== Installing the Spark operator
```

Comment:
I'm missing a note about nfs here

Reply:
We need to add a doc or a note here to install nfs first.

Reply:
Yea, at this point it is too late, I think we mentioned a couple of time the need for an RWM storage class when we install Cloudflow.

Reply:
I have added instructions about installing NFS.